pkg/oss/nodeserver.go

```diff
@@ -225,10 +226,14 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 			return nil, errors.New("Mount is failed, with create path err: " + err.Error() + mountPath)
 		}
 
-		metaZoneID := GetMetaDataAsync(regionTag)
-		if strings.Contains(opt.URL, metaZoneID) && !strings.Contains(opt.URL, "internal") && !utils.IsPrivateCloud() {
+		metaRegionID := GetRegionID()
```

Comment:
We may get region ID once on initialization, instead of getting it every time on `NodePublishVolume`. The timeout (2s) may be too short, but it is a waste of time on non-ECS servers.

Or better, we may extract it so that it is independent of any driver, multiple drivers would need region ID.

That said, the bug fix is valid. Refactoring can be done afterward.